testsuite/dependencies/init.yml

```diff
@@ -0,0 +1,20 @@
+archive: data.zip
+pretest_test_cases:
+- &p1 {in: 1.txt, out: 4.txt}
+- &p2 {in: 2.txt, out: 1.txt}
+- &p3 {in: 3.txt, out: 10.txt, dependencies: [*p1]}
+test_cases:
+- &m1 {in: 5.txt, out: 16.txt, points: 1, dependencies: [*p1, *p3]}
+- &m2
+  batched:
+  - {in: 4.txt, out: 2.txt}
+  - {in: 6.txt, out: 3.txt}
+  points: 1
+  dependencies: [*p2]
```

Comment:
Weak preference to naming this node `depends`, `dependencies` is pretty hard to type over and over by hand.

Reply:
I don't really care either way, to be honest.